app/jobs/dttp_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DttpJob < ApplicationJob
```

Comment:
I think we'll need to be more explicit with the naming of this. We'll have create, update, check for TRN and probably others that all interact with DTTP.

Reply:
I was holding back until the next jobs needed to be created, but happy to be more explicit now.